polaris.shopify.com/src/styles/globals.scss

```diff
@@ -103,25 +103,36 @@
   --focus-outline-inside: inset 0 0 0 2px var(--text-strong);
 }
 
-* {
+// Resets for both the site and polaris examples
+body,
+html {
   margin: 0;
   padding: 0;
-  color: inherit;
-  font-family: inherit;
-  font-weight: inherit;
-  font-size: 100%;
-  line-height: inherit;
-  text-decoration: none;
-  box-sizing: border-box;
-  list-style-type: none;
-  -webkit-font-smoothing: antialiased;
-  border: none;
-  appearance: none;
-  outline: none;
 }
 
-*:focus-visible {
-  box-shadow: var(--focus-outline);
+// Resets and styles that should not affect Polaris examples
+.non-polaris-styles {
```

Comment:
Love this change the `non-polaris-styles` class name is a little confusing. Could we call this `.exclude-example-styles` or something that explicitly describes what it is doing?

Reply:
I changed it to `styles-for-site-but-not-polaris-examples` 🤪